File: Baekjoon_Online_Judge/NO_11908/11908.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 07                   | #
# | 11908 카드                | #
# ---------------------------- #


if __name__ == "__main__":
    n = int(r_input())
    c = list(map(int, r_input().split()))

    print(sum(c) - max(c))

File: Baekjoon_Online_Judge/NO_11908/11908.c
#include <stdio.h>
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 07                   | 
 | 11908 카드                | 
 ---------------------------- 
*/

int main(){
    int n, total = 0, maxi = 0;

    scanf("%d", &n);

    for(int i = 0; i < n; i ++){
        int c;

        scanf("%d", &c);
        total += c;

        if (maxi < c) maxi = c;
    }

    printf("%d", total - maxi);
    
    return 0;
}
